### PR TITLE
Add spec to get disk type

### PR DESCRIPTION
**Pull request by smalleni**
This helps us know if a disk is rotational or SSD.

### CODE REVIEW

File: insights/specs/default.py [hunk 1]
```diff
@@ -180,6 +180,7 @@ def is_ceph_monitor(broker):
     dirsrv = simple_file("/etc/sysconfig/dirsrv")
     dirsrv_access = glob_file("var/log/dirsrv/*/access")
     dirsrv_errors = glob_file("var/log/dirsrv/*/errors")
+    disk_type = glob_file("/sys/block/*/queue/rotational")
```
Comment (bfahr) [inline insights/specs/default.py]: @smalleni you also need to add a RegistryPoint for the `disk_type` spec to the file `insights/specs/__init__.py`.

**Comment by bfahr** [comment]
@smalleni are you adding a parser for this new spec?

**Comment by smalleni** [comment]
@bfahr Not right now, Is it required to go along with the spec in this PR?

**Comment by bfahr** [comment]
@smalleni have you had a chance to work on a parser related to your spec changes?

**Comment by smalleni** [comment]
@bfahr I think we can close this as the same info was available through lsblk. Doesn't look like this is needed.

**Comment by bfahr** [comment]
Closing PR as data is available from other sources.

**Pull request closed without merging**